framework/src/test/java/tests/ReturnsReceiverLombokTest.java

```diff
@@ -0,0 +1,38 @@
+package tests;
+
+import java.io.File;
+import java.util.List;
+import org.checkerframework.common.returnsreceiver.ReturnsReceiverChecker;
+import org.checkerframework.framework.test.CheckerFrameworkPerDirectoryTest;
+import org.junit.runners.Parameterized.Parameters;
+
+/**
+ * tests the returns receiver checker's lombok integration, the test files in
```

Comment:
You should start javadoc sentences with an upper case letter and end it with a `.`.